packages/office-ui-fabric-react/src/components/Conformance.test.tsx

```diff
@@ -0,0 +1,219 @@
+import * as React from 'react';
```

Comment:
Can this live somewhere other than components?

Reply:
I based these tests on ComponentExamples.test.tsx - should those be moved as well?

Reply:
Yeah, just trying to alleviate confusion as people dive in and wonder what that component does.